e2e/windows/e2e-helper.sh

```diff
@@ -68,19 +68,18 @@ upload_linux_file_to_storage_account() {
     E2E_MC_RESOURCE_GROUP_NAME="MC_${E2E_RESOURCE_GROUP_NAME}_${AZURE_E2E_CLUSTER_NAME}_$AZURE_BUILD_LOCATION"
     MC_VMSS_NAME=$(az vmss list -g $E2E_MC_RESOURCE_GROUP_NAME --query "[?contains(name, 'nodepool')]" -ojson | jq -r '.[0].name')
     VMSS_INSTANCE_ID="$(az vmss list-instances --name $MC_VMSS_NAME -g $E2E_MC_RESOURCE_GROUP_NAME | jq -r '.[0].instanceId')"
+    VMSS_RESOURCE_ID="$(az vmss show --name $MC_VMSS_NAME --resource-group $E2E_MC_RESOURCE_GROUP_NAME --instance-id $VMSS_INSTANCE_ID | jq '.id')"
+    az vmss identity assign --identities "${AZURE_MSI_RESOURCE_STRING}" -g $E2E_MC_RESOURCE_GROUP_NAME -n $MC_VMSS_NAME
+    az vmss update-instances -g $E2E_MC_RESOURCE_GROUP_NAME -n $MC_VMSS_NAME --instance-ids "*"
```

Comment:
updating instance should take a long time. Is it possible to create the vmss with setting the identity?

Reply:
In my test, updating instance doesn't take too much time. But I think I could take a try to create the vmss with setting the identity.